tools/api-markdown-documenter/src/test/HtmlEndToEnd.test.ts

```diff
@@ -62,7 +67,10 @@ const testConfigs = new Map<
 			includeTopLevelDocumentHeading: true,
 			hierarchy: HierarchyConfigurations.sparse,
 			minimumReleaseLevel: ReleaseTag.Public, // Only include `@public` items in the docs suite
-			skipPackage: (apiPackage) => apiPackage.name === "test-suite-b", // Skip test-suite-b package
+			exclude: (apiItem) =>
+				// Skip test-suite-b package
+				apiItem.kind === ApiItemKind.Package &&
+				(apiItem as ApiPackage).name === "test-suite-b",
```

Comment:
Follow-up suggestion: this makes me think an `isPackage()` type guard (maybe similar ones for common cases?) would be nice.

Reply:
I've avoided that thus far since presumably we would want one for each kind of API item. But packages are a slightly more special case, and possibly a more common case for filtering and such.